roles/zooplus/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- include: globalprotect.yml
+  vars:
+    globalprotect_package: GlobalProtect_deb-5.1.0.0-101.deb
```

Comment:
Maybe you want to remove the package version here. 
Apt don't check the file name and you want always latest.

Reply:
once I can move to apt, I will remove this shit :P